### PR TITLE
Disabling CronJobs default at runtime

### LINKED ISSUE #1806: Disable our Quartz cron tasks
# Task Description

We need to disable Quartz, so that it doesn't start up the cron tasks below. At the moment we have bigger issues to debug and this is getting the way. We will have a follow-up task at a later time to re-enable these.

Optimally, this should be controllable via a property in the `strongbox.yaml`. Quartz provides a property for this as well.

### List of Cron jobs to be disabled at runtime
* [x] `org.carlspring.strongbox.cron.jobs.DownloadRemoteFeedCronJob`
* [x] `org.carlspring.strongbox.cron.jobs.DownloadRemoteMavenIndexCronJob`
* [x] `org.carlspring.strongbox.cron.jobs.FetchRemoteNpmChangesFeedCronJob`

# Task Relationships

This task:
* Relates to: #1649

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @sbespalov
  * @steve-todorov

### PR DESCRIPTION
**Pull request by anki2189**
# Pull Request Description

This pull request closes #1806 

# Acceptance Test

* [x] Building the code with `mvn clean install -Dintegration.tests` still works.
* [x] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [x] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [x] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.

# Questions

* Does this pull request break backward compatibility? 
  * [ ] Yes
  * [x] No

* Does this pull request require other pull requests to be merged first? 
  * [ ] Yes, please see #...
  * [x] No

* Does this require an update of the documentation?
  * [ ] Yes, please see strongbox/strongbox-docs#{PR_NUMBER}
  * [x] No

### CODE REVIEW

File: strongbox-web-core/pom.xml [hunk 1]
```diff
@@ -113,6 +113,9 @@
                         <strongbox.basedir>${project.build.directory}</strongbox.basedir>
                         <logging.config>${project.build.directory}/strongbox/etc/logback-spring.xml</logging.config>
                         <logging.debug>true</logging.debug>
+                        <strongbox.npm.remote.changes.enabled>false</strongbox.npm.remote.changes.enabled>
+                        <strongbox.nuget.download.feed>false</strongbox.nuget.download.feed>
+                        <strongbox.download.indexes>false</strongbox.download.indexes>
```
Comment (carlspring) [inline strongbox-web-core/pom.xml:118]: You should also disabled them in the distribution as well.

Comment (anki2189) [reply strongbox-web-core/pom.xml:118]: For distribution I am thinking to disable it from `setenv.conf `. Am I on right path @steve-todorov ?

Comment (steve-todorov) [reply strongbox-web-core/pom.xml:118]: Yes, that's correct. Better to have them disabled for now in the distribution as well. :)

Comment (sbespalov) [reply strongbox-web-core/pom.xml:118]: > For distribution I am thinking to disable it from setenv.conf . Am I on right path @steve-todorov ?

right, as I can see it disabled

**Comment by carlspring** [comment]
@anki2189 ,

Could you please rebase this and also create a pull for it against the `issue-1649` branch? Thanks! :)

**Comment by sbespalov** [comment]
> @anki2189 ,
> 
> Could you please rebase this and also create a pull for it against the `issue-1649` branch? Thanks! :)

it would be probably better to rebase the `issue-1649` to avoid conflicts.

**Comment by carlspring** [comment]
I think it would be better to have two one pull request for each of the two branches, so that we can have this disabled for both branches.

**Comment by sbespalov** [comment]
> I think it would be better to have one pull request for each of the two branches, so that we can have this disabled for both branches.

in this case there will be conflicts in `issue-1649` against `master`. My suggestion was to merge this PR and then rebase  `issue-1649` on `master`.

**Comment by carlspring** [comment]
Okay, that works for me as well.

**Pull request merged**